spec/unit/facter/util/fact_rabbitmq_nodename_spec.rb

```diff
@@ -28,6 +28,16 @@
       end
     end
 
+    context 'with dash in username' do
```

Comment:
this should be 'nodename', no?

Reply:
Yes, thanks for the hint. Will be done soon.